### DISCUSSION TITLE
Is it possible to collect data from JK-BMS Inverter PB series in parallel?

### DISCUSSION THREAD

**Discussion by gyrex** [syssi/esphome-jk-bms#595] [Q&A]
I have 4 JK-BMS PB series BMSs in parallel with the master on ID0. I can use the JK software to connect to the battery bank in parallel  mode using RS485 and see and control the 4 batteries. Is there any way to do this using esphome-jk-bms? I'd much prefer to see all the data in home assistant rather than using their clunky software on a PC.

**Comment by moto501** [comment]
looking for the same answer

**Reply by gyrex** [reply]
I ended up using this fork: https://github.com/txubelaxu/esphome-jk-bms/blob/main/components/jk_rs485_bms/README.md

and it works really well.

**Reply by moto501** [reply]
yup, tested that last night and works well, thanks! ..

Question though, if you run the ESP as master and two batteries as slaves, do you lose any functionality? Should you have a JK BMS as master as a general rule? I'm a bit unclear on this. Ideally, i would prefer to control all my BMS from a single interface and ditch the BLE per BMS etc, but just need to know if i make the ESP the master that the 'system' will behave etc. TIA
